packages/exp/src/expo_commands/build-webhook.js

```diff
@@ -0,0 +1,108 @@
+import { URL } from 'url';
+import { BuildWebhook, Exp } from 'xdl';
+import chalk from 'chalk';
+
+import log from '../log';
+
+export default program => {
+  program
+    .command('build-webhook:set [project-dir]')
+    .option('--url <webhook-url>', 'Webhook to be called after building the app.')
+    .option(
+      '--secret <webhook-secret>',
+      'Secret to be passed as a query param (?secret=<webhook-secret>) to the Webhook. It has to be at least 16 characters long.'
+    )
+    .description(`Set the webhook for project to be called after finishing a standalone app build.`)
+    .asyncActionProjectDir(async (projectDir, options) => {
+      _ensureOptionsAreValid(options);
+
+      const { args: { remoteFullPackageName: experienceName } } = await Exp.getPublishInfoAsync(
+        projectDir
+      );
+
+      log(`Setting webhook and secret for ${experienceName}`);
+
+      try {
+        const webhook = {
+          url: options.url,
+          secret: options.secret,
+        };
+        await BuildWebhook.setBuildWebhookAsync(experienceName, webhook);
+      } catch (e) {
+        log.error(e);
+        throw new Error('Unable to set webhook and secret for this project.');
+      }
+
+      log('All done!');
+    }, true);
+
+  program
+    .command('build-webhook:fetch [project-dir]')
```

Comment:
I propose following naming for the commands:
- `expo webhooks:set --event build --url <url>`: set the build webhook
- `expo webhooks:show`: show the currently configured webhook(s)

This way, if we ever add more events that can trigger webhooks, we don't need a new global command name for each of them. It also avoids having `-` in the command name.

Also changed the other command from `fetch` to `show` to be consistent with `push:android:show`.

Reply:
👍 will change that